test/common/testlib.py

```diff
@@ -1141,13 +1141,23 @@ def start_machine_troubleshoot(
         known_host: bool = False,
         password: str | None = None,
         expect_closed_dialog: bool = True,
+        expect_warning: bool = True,
+        need_click: bool = True
```

Comment:
that sounds a bit generic, maybe `expect_curtain`?

Reply:
Yes, good idea! Done.